packages/loopring_v3.js/src/protocol_v3.ts

```diff
@@ -0,0 +1,232 @@
+import BN from "bn.js";
+const fs = require("fs");
+import Web3 from "web3";
+
+interface StakeAmounts {
+  exchangeStake: BN;
+  protocolFeeStake: BN;
+}
+
+/**
+ * Processes all data of the Loopring v3 protocol contract.
+ */
+export class ProtocolV3 {
```

Comment:
I guess I still need to figure out how this class shall be used. We should probably populate data into a database. I'll keep reading the other code and get back to this comment.

Reply:
Maybe some of the code can be done elsewhere, but what the code does needs to be done somewhere. Doing the processing of the data here as well made sense to me, but we'll see.